.node-image

```diff
@@ -0,0 +1 @@
+node:12-bullseye-slim
```

Comment:
I'm not sure adding all these one-off version files is great. In this case it might be fine to just stick it in `run-in-node-container.sh` since we should only need node at build-time and we can do that all with that script AFAICT.

Originally I thought we might need to do some caching, but gitignored `node_modules` seems to handle that fine.

regarding the choice of image:
- relatively small, but complete
- we are on node 12.X under bazel currently

Reply:
I like the idea of sticking in in `run-in-node-container.sh`, compilation for `deck` can also be done in a way similar to `yarn install`, we should be fine

Reply:
I'll do this.

Reply:
done